### PR TITLE
Api extended rita3

### PR DESCRIPTION
**Pull request by RiTiS7**

### CODE REVIEW

File: server/controllers/books.js [hunk 1]
```diff
@@ -1,52 +1,54 @@
...
+  index: async (req, res) => {
     const books = await Book.find({}, '-_id');
     res.status(200).json(books);
   },
 
-  newBook: async (req, res, next) => {
+  newBook: async (req, res) => {
     const newBook = new Book(req.body);
-    try{
+    try {
```
Comment (Alaev) [inline server/controllers/books.js]: add comment for try catch

File: server/controllers/books.js [hunk 2]
```diff
@@ -1,52 +1,54 @@
...
     const { bookISBN } = req.params;
     const newBook = req.body;
-    const replaced = await Book.findOneAndUpdate({ 'ISBN': bookISBN }, newBook, {new: true});
+    const replaced = await Book.findOneAndUpdate({ ISBN: bookISBN }, newBook, { new: true });
 
     if (!replaced) {
-      return res.status(404).json({ message: 'Can\'t replaced book, book was not found' });
+      res.status(404).json({ message: "Can't replaced book, book was not found" });
+    }else{
+      res.status(200).json({ replaced, message: 'Great news, book was replaced' });
```
Comment (Alaev) [inline server/controllers/books.js]: why do we need a replaced here?

Comment (RiTiS7) [inline server/controllers/books.js]: To test if the changes were made accordingly and to show the user the updated book

File: server/controllers/books.js [hunk 3]
```diff
@@ -2,48 +2,40 @@ const Book = require('../models/Books');
...
+    const books = await Book.find({}, '-_id -__v');
     res.status(200).json(books);
   },
 
   newBook: async (req, res) => {
     const newBook = new Book(req.body);
-    await newBook.save();
-
-    res.status(201).json({ message: 'Great news, Book was added to Library' });
+    try {
```
Comment (Alaev) [inline server/controllers/books.js]: add try comment here about try catch use and I will approve pull request.

**Comment by Alaev** [comment]
Going to merge please be sure to add the comment about try catch

**Pull request merged**